tests/test_conf.py

```diff
@@ -9,6 +9,11 @@
 from ralph.conf import CommaSeparatedTuple, Settings, settings
 from ralph.exceptions import ConfigurationException
 
+# import os
+# def test_env_dist(fs, monkeypatch):
+#     fs.create_file(".env", contents=os.read("../.env.dist"))
+#     Settings()
+
```

Comment:
Don't forget about this too!

Reply:
oups. yes this is for a separate PR

Reply:
see: https://github.com/openfun/ralph/issues/462